### PR TITLE
Trivia: Support pausing and resuming games

### LINKED ISSUE #6398: Implement /trivia pause

### PR DESCRIPTION
**Pull request by AnnikaCodes**
Fixes #6398.

### CODE REVIEW

File: server/chat-plugins/trivia.ts [hunk 1]
```diff
@@ -1726,7 +1763,9 @@ const commands: ChatCommands = {
 			`- <code>/ta [answer]</code> - Answer the current question.`,
 			Utils.html`- <code>/trivia kick [username]</code> - Disqualify a participant from the current trivia game. Requires: % @ # &`,
 			`- <code>/trivia leave</code> - Makes the player leave the game.`,
-			Utils.html`- <code>/trivia end</code> - End a trivia game. Requires: + % @ #`,
+			Utils.html`- <code>/trivia end</code> - End a trivia game. Requires: + % @ # &`,
+			Utils.html`- <code>/trivia pause</code> - Pauses a trivia game. Requires: + % @ # &`,
+			Utils.html`- <code>/trivia resume</code> - Resumes a paused trivia game. Requires: + % @ # &`,
 			``,
 			`<strong>Question-modifying commands:</strong>`,
 			Utils.html`- <code>/trivia submit [category] | [question] | [answer1], [answer2] ... [answern]</code> - Adds question(s) to the submission database for staff to review. Requires: + % @ # &`,
```
Comment (Zarel) [inline server/chat-plugins/trivia.ts:1771]: Hm, none of these actually require `Utils.html`.

```suggestion
			`- <code>/trivia end</code> - End a trivia game. Requires: + % @ # &`,
			`- <code>/trivia pause</code> - Pauses a trivia game. Requires: + % @ # &`,
			`- <code>/trivia resume</code> - Resumes a paused trivia game. Requires: + % @ # &`,
			``,
			`<strong>Question-modifying commands:</strong>`,
			`- <code>/trivia submit [category] | [question] | [answer1], [answer2] ... [answern]</code> - Adds question(s) to the submission database for staff to review. Requires: + % @ # &`,
```

We should probably standardize on just not having it?

File: server/chat-plugins/trivia.ts [hunk 2]
```diff
@@ -1726,7 +1763,9 @@ const commands: ChatCommands = {
 			`- <code>/ta [answer]</code> - Answer the current question.`,
 			Utils.html`- <code>/trivia kick [username]</code> - Disqualify a participant from the current trivia game. Requires: % @ # &`,
```
Comment (Zarel) [inline server/chat-plugins/trivia.ts:1764]:
```suggestion
			`- <code>/trivia kick [username]</code> - Disqualify a participant from the current trivia game. Requires: % @ # &`,
```

(If you could also edit the two instances above where I can currently suggest.)

**Comment by Zarel** [comment]
Everything else looks good, though.

**Comment by AnnikaCodes** [comment]
(The failing check is seemingly unrelated to my changes, and checks pass locally.)

**Comment by Zarel** [comment]
Yeah, the failing check is because of a botched merge for `/sendhtmlpage`.

**Pull request merged**